thriftcli/thrift_cli.py

```diff
@@ -65,6 +65,16 @@ def run(self, method_name, request_body, return_json=False):
         """
         request_args = self._thrift_argument_converter.convert_args(self._service_reference, method_name, request_body)
         result = self._thrift_executor.run(method_name, request_args)
+        if not request_body:
```

Comment:
Thrift methods are not required to take parameters, so this shouldn't necessarily raise an error.
Perhaps add some check of `request_args` to distinguish the error cases?

Reply:
ill close this. I have a better idea to achieve what I was going for here. If I add logging I can add a verbose option that would just print out "sending the following request" which will provide the hint